config/features/flags.go

```diff
@@ -135,6 +135,10 @@ var (
 		Name:  "enable-verbose-sig-verification",
 		Usage: "Enables identifying invalid signatures if batch verification fails when processing block",
 	}
+	EnableCapellaEngineMethods = &cli.BoolFlag{
+		Name:  "enable-capella-engine-methods",
```

Comment:
I think `enable-optional-engine-methods` might be better here, as it more accurately indicates what this flag is for. We have capella specific engine methods that will not use this flag